rs/consensus/src/idkg/payload_verifier.rs

```diff
@@ -1143,6 +1149,55 @@ mod test {
         );
     }
 
+    #[test]
```

Comment:
I'm not sure I understand this test fully.

If I understand correctly, we are testing that `validate_new_signature_agreements` ignores `signature_request_contexts` of vetkd requests, but since we can not directly check that, we insert a vetkd context with `pseudo_random_id` and then check that the function does not return that contextr (since it only returns idkg contexts).

If this is the case, I think it might sense to describe this a little bit better, because it is not so obvious.

Also, we might want to rename `signature_request_contexts` to `idkg_signature_request_contexts`, to drive the point home